rxbindings/src/test/java/com/amplifyframework/rx/RxApiBindingTest.java

```diff
@@ -225,12 +228,20 @@ public void subscribeStartsEmitsValuesAndCompletes() throws InterruptedException
         );
 
         // Act: subscribe via binding
-        TestObserver<GraphQLResponse<Model>> observer =
-            rxApi.subscribe(request).test();
+        RxApiBinding.RxSubscriptionOperation<GraphQLResponse<Model>> rxOperation = rxApi.subscribe(request);
```

Comment:
I'd add an `import ... ` for `RxSubscriptionOperation..` directly just to cut down on some of this boiler plate a tiny bit.